client/state/index.js

```diff
@@ -64,6 +65,7 @@ export const reducer = combineReducers( {
 	preview,
 	posts,
 	postTypes,
+	productsList,
```

Comment:
Including this in reducer reveals that the default state for `productsList.items` (`null`) is invalid, causing a warning to be shown in the console.